### PR TITLE
Licence for Dilithium and Kyber

### PR DESCRIPTION
**Pull request by vincentvbh**
As titled.
At the beginning of each file, you will find the license information.

#### Manually checked properties
* [ ] Generated Github workflow (run ``.github/workflows/generate_workflows.py``) (new schemes)
* [ ] No stringification macros
* [ ] Output-parameter pointers in functions are on the left
* [ ] Negative return values on failure of API functions (within restrictions of FO transform).
* [ ] variable declarations at the beginning (except in `for (size_t i=...`)
* Optional:
  * [ ] All integer types are of fixed size, using `stdint.h` types (including `uint8_t` instead of `unsigned char`)
  * [ ] Integers used for indexing are of size `size_t`

### CODE REVIEW

File: crypto_kem/kyber1024/aarch64/api.h [hunk 1]
```diff
@@ -1,3 +1,11 @@
+
+/*
+ * This file is licensed
+ * under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.html) or
```
Comment (dstebila) [inline crypto_kem/kyber1024/aarch64/api.h:4]: Some of the files are MIT, some are Apache 2, is there a reason?

File: crypto_kem/kyber1024/aarch64/indcpa.c [hunk 2]
```diff
@@ -1,15 +1,50 @@
...
-#include "ntt.h"
+
+/*
+ * This file was originally licensed
+ * under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.html)
+ * at https://github.com/GMUCERG/PQC_NEON/blob/main/neon/kyber or
+ * public domain at https://github.com/cothan/kyber/blob/master/neon
+ *
+ * We choose
+ * CC0 1.0 Universal or the following MIT License for this file.
```
Comment (dstebila) [inline crypto_kem/kyber1024/aarch64/indcpa.c:9]: This is also really confusing to me. Are you trying to say it's triple licensed?

File: crypto_kem/kyber1024/aarch64/api.h (hunk 1 above)
Comment (vincentvbh) [reply crypto_kem/kyber1024/aarch64/api.h:4]: We didn't change this file. The licensing is from where we got the file.

File: crypto_kem/kyber1024/aarch64/indcpa.c (hunk 2 above)
Comment (vincentvbh) [reply crypto_kem/kyber1024/aarch64/indcpa.c:9]: No, we license it under CC0 and MIT. The first lines say where the unmodified file comes from.

Comment (vincentvbh) [reply crypto_kem/kyber1024/aarch64/indcpa.c:9]: For clarification, the first lines say where the file came from before modification. We modify and license it under CC0 and MIT.

**Comment by mkannwischer** [comment]
Can we merge this or do we need to change it somehow? I would not know how to do it any better.

**Review by dstebila — APPROVED** [review]

**Pull request merged**